### DISCUSSION TITLE
How to enable Contributors plugin?

### DISCUSSION THREAD

**Discussion by tiziodcaio** [vuepress/core#204] [Q&A] [answered]
I tried without success
Environment:
- Github
- Deploying to github pages (--depth 0)
- Vuepress 2.0.0-beta.17

**Comment by meteorlxy** [comment, marked as answer]
Yes, it's a known bug and we didn't find time to figure it out - the `git shortlog` command does not work well in CI enviroment.

You can transform it into a Bug Report issue

**Reply by tiziodcaio** [reply]
Ok super!
